app/Notifications/Mship/Endorsement/EndorsementRequestCreated.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Notifications\Mship\Endorsement;
+
+use App\Models\Mship\Account\EndorsementRequest;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class EndorsementRequestCreated extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    /**
+     * Create a new notification instance.
+     */
+    public function __construct(private EndorsementRequest $endorsementRequest)
+    {
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @return array<int, string>
+     */
+    public function via(object $notifiable): array
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     */
+    public function toMail(object $notifiable): MailMessage
+    {
+        return (new MailMessage)
+            ->from(config('mail.from.address'), 'VATSIM UK - Training Department')
+            ->subject('Endorsement Request Created')
+            ->view('emails.mship.endorsement.endorsement_request_created', [
+                'recipient' => $notifiable,
+                'requestIndexUrl' => url('/admin/endorsement-requests'),
```

Comment:
Can we use the route helper to ensure that if the URI changes this updates too?

Reply:
Sadly filament is only seen as a single 'route' (`filament.update`), so a bit of hard coding is needed here.

Reply:
Only just saw this, but fwiw should be able to do something like
```php
                'requestIndexUrl' => EndorsementRequestResource::getUrl('index'),
```